Doc/library/typing.rst

```diff
@@ -1954,7 +1959,7 @@ without the dedicated syntax, as documented below.
 
    .. doctest::
 
-      >>> from typing import ParamSpec
+      >>> from typing import ParamSpec, get_origin
```

Comment:
I don't mind the change you're making here. But the reason why this doesn't currently fail when Sphinx runs our doctests in CI is because of this, which is run before every doctest block:

https://github.com/python/cpython/blob/d49aba5a7a3c695213810a9f82715809c6332df2/Doc/library/typing.rst?plain=1#L5-L9